### ISSUE TITLE
use drop_out=0.2 in the inference mode, isn't wrong way?

### ISSUE THREAD

**Issue by PythonImageDeveloper** [qubvel/efficientnet#61]
Hi 
In the efficient-net model used dropout layer in the final prediction layer and also used in the Conv layers with rate 0.2 for training mode, Why in the inference mode didn't set rate of these layers to 1.0, and these layers don't operate in the inference mode?

**Comment by qubvel** [comment]
Keras automatically disable dropout during inference stage

**Comment by PythonImageDeveloper** [comment]
@qubvel 
That means, when we use model.predict the keras automatically put the dropout and BN layers in the inference phase?

**Comment by qubvel** [comment]
No, it is default keras behavior for a long time
